### DISCUSSION TITLE
DS4 support in Linux

### DISCUSSION THREAD

**Discussion by Arbitrate3280** [LizardByte/community#190] [Sunshine] [answered]
Hi! I would like to know if it's possible to add DS4 support to linux. I have a DS4 and would like it to be recognized as such. Having it also available in linux would make even more sense, seeing as ds4 support is already baked right into the kernel.

**Comment by ReenigneArcher** [comment, marked as answer]
Linux now supports DS5, Xbox One, and Nintendo Switch controllers. This is available in beta/pre-releases, and will be available in the next stable release as well.
